test/config.test.js

```diff
@@ -1303,7 +1303,8 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
             },
```

Comment:
Update all tests to check for `sub` and `iss`, and also to check that the JWT itself doesn't contain the _jwtPName.

Reply:
Done